### PR TITLE
If hud is disabled don't show lua error messages

### PR DESCRIPTION
**Pull request by DanielSvoboda**
Don't draw MenuDrawLuaErrors when hud is off in cl_drawhud

### CODE REVIEW

**Comment by Kefta** [comment]
Please cache the convar object outside of the hook and use GetBool on it.

**Comment by DanielSvoboda** [comment]
Everything fixed :)

**Comment by Kefta** [comment]
Just cache the convar object and call GetBool on demand, don't cache GetBool's result.

File: garrysmod/lua/menu/errors.lua [hunk 1]
```diff
@@ -55,9 +55,12 @@ end )
 
 local matAlert = Material( "icon16/error.png" )
 
+local cl_drawhud = GetConVar("cl_drawhud")
+
 hook.Add( "DrawOverlay", "MenuDrawLuaErrors", function()
 
 	if ( table.IsEmpty( Errors ) ) then return end
+	if (not cl_drawhud:GetBool() ) then return end
```
Comment (Kefta) [inline garrysmod/lua/menu/errors.lua:63]: if ( !cl_drawhud:GetBool() ) then

Needs the space and use C ops.

**Review by Kefta — CHANGES_REQUESTED** [review]

**Comment by DanielSvoboda** [comment]
Done

File: garrysmod/lua/menu/errors.lua (hunk 1 above)
Comment (DanielSvoboda) [reply garrysmod/lua/menu/errors.lua]: Done

**Comment by Kefta** [comment]
Spacing is still wrong. There should be a space between the parenthesis and the not operator and no space between the op and the statement.

**Review by Kefta — APPROVED** [review]

**Review by robotboy655 — APPROVED** [review]

**Pull request merged**